contracts/{{adapter_name}}/src/bin/publish.rs

```diff
@@ -30,10 +30,11 @@ fn publish(networks: Vec<ChainInfo>) -> anyhow::Result<()> {
         // Create an [`AbstractClient`]
         let abstract_client: AbstractClient<Daemon> = AbstractClient::new(chain.clone())?;
 
-        // Get the [`Publisher`] that owns the namespace, otherwise create a new one and claim the namespace
-        let publisher: Publisher<_> = abstract_client
-            .publisher_builder(adapter_namespace)
-            .build()?;
+        // Get the [`Account`] that owns the namespace, otherwise create a new one and claim the namespace
+        let publisher_acc = abstract_client.fetch_or_build_account(adapter_namespace, |builder| builder.namespace(Namespace::from_id({{adapter_name | shouty_snake_case}}_ID).unwrap()))?;
```

Comment:
Would be nice to just clone namespace, but "builder fn" doesn't capture environment, as it's function instead of closure, fixed in https://github.com/AbstractSDK/abstract/pull/519